ci/jenkins/test.sh

```diff
@@ -670,6 +689,177 @@ function clean_tmp() {
     find ${WORKDIR} -name "support-bundles*" -mtime +7 -exec rm -rf {} \; 2>&1 | grep -v "Permission denied" || true
 }
 
+# redeploy_k8s redeploys K8s cluster when flexible-ipam is set and existing cluster ip-mode doesn't match input ip-mode
+function redeploy_k8s() {
```

Comment:
As we redeploy only when IP mode does not match, probably we should rename the func to redeploy_k8s_if_ip_mode_changes(), or ensure_k8s_ip_mode()?

Reply:
`redeploy_k8s_if_ip_mode_changes` is OK. Fixed.